packages/web-client/app/components/card-pay/deposit-workflow/index.hbs

```diff
@@ -1,26 +1,92 @@
-  <h3>{{this.workflow.name}}</h3>
-  {{#each this.workflow.milestones as |milestone i|}}
-    {{#let (or milestone.complete (lte i this.workflow.activeMilestoneIndex)) as |displayMilestone|}}
-      {{#if displayMilestone}}
-        {{#each milestone.postables as |postable j|}}
-          <div class="thread-message" data-test-thread-message={{j}} data-test-milestone={{i}}>
-            <div data-test-author-name>
-              {{postable.author.name}}
-            </div>
+<Boxel::Thread class="deposit-workflow">
+  <:header>
+    <Boxel::ThreadHeader @title={{this.workflow.name}} />
+  </:header>
+  <:content>
+    <Boxel::ExpandableBanner
+      class="deposit-workflow__banner"
+      @icon="deposit"
+      @summary="Deposit Funds"
+    >
+      <p class="deposit-workflow__banner-paragraph">
+        As a Supplier, you deposit funds from your Ethereum mainnet wallet into the
+        CARD Protocol's reserve pool and receive the corresponding amount of CPXD
+        tokens in your xDai chain wallet.
+      </p>
+      <p class="deposit-workflow__banner-paragraph">
+        We currently support two types of asset you can deposit into the reserve pool:
+        <ul class="deposit-workflow__banner-list">
+          <li>Dai stablecoins (DAI)</li>
+          <li>Cardstack ERC-20 tokens (CARD)</li>
+        </ul>
+      </p>
+      <p class="deposit-workflow__banner-paragraph">
+        The CARD Protocol's reserve pool facilitates payments between customers
+        and merchants on the Cardstack network.
+      </p>
+      <p class="deposit-workflow__banner-small-text">
+        Fees: Suppliers earn transaction fees of 0.5% for all transactions across
+        the CARD Protocol. Those fees will be distributed to Suppliers based on
+        their proportional share of supplied tokens in the reserve pool, averaged
+        over the 30-day period in which the transactions occurred. {{!-- <a href="#">More details here.</a> --}}
+        {{!-- TODO: add link --}}
+      </p>
+    </Boxel::ExpandableBanner>
+    {{#each this.workflow.milestones as |milestone i|}}
+      {{#let (or milestone.complete (lte i this.workflow.completedMilestoneCount)) as |displayMilestone|}}
+        {{#if displayMilestone}}
+          {{#each milestone.visiblePostables as |postable j|}}
             {{#if postable.message}}
-              {{format-workflow-message postable.message}}
+              <Boxel::ThreadMessage
+                class="deposit-workflow__thread-message"
+                @datetime={{postable.timestamp}}
+                @imgURL={{postable.author.imgURL}}
+                @hideMeta={{postable-meta-identical postable (object-at milestone.visiblePostables (dec j))}}
+                data-test-postable={{j}} data-test-milestone={{i}}
+              >
+                {{format-workflow-message postable.message}}
```

Comment:
![image](https://user-images.githubusercontent.com/39579264/114397773-9f6b1f80-9bd1-11eb-85cf-43ddf0b2fd32.png)

![image](https://user-images.githubusercontent.com/39579264/114398210-1dc7c180-9bd2-11eb-81d1-f4d80fbc7842.png)

![image](https://user-images.githubusercontent.com/39579264/114397715-8e221300-9bd1-11eb-81ec-bc6798c21c40.png)

![image](https://user-images.githubusercontent.com/39579264/114398278-31732800-9bd2-11eb-8790-64e9747ed0f6.png)

Some browser inconsistencies with list styling - top is Safari, bottom is Chrome. I think using user-agent defaults is ok for now, but ideally we should keep the reset we had before and scope the styles for the `format-workflow-message` styles to a "wrapper" selector, eg. `.wrapper ul`.

Reply:
What do you think about a "prose" or "standard-formatting" utility class that we apply to elements that contain markdown-rendered content.

Reply:
I think that fits nicely!

Reply:
✅